### PR TITLE
Fix bug where bottom bar bindings are done when the settings menu isn't visible

### PR DESCRIPTION
**Pull request by kyle-emmerich**

### CODE REVIEW

File: CoreScriptsRoot/Modules/Settings/SettingsHub.lua [hunk 1]
```diff
@@ -559,7 +559,7 @@ local function CreateSettingsHub()
 			--to update, or we can just avoid using it so soon.
 			RunService.Heartbeat:wait()
 
-			if shouldShowBottomBar() then
+			if this.Visible and shouldShowBottomBar() then
```
Comment (hockeychaos) [inline CoreScriptsRoot/Modules/Settings/SettingsHub.lua]: should this check be in shouldShowBottomBar?

Comment (kyle-emmerich) [inline CoreScriptsRoot/Modules/Settings/SettingsHub.lua]: I thought about it but I don't think it makes sense. You probably shouldn't even be asking if you can show the bottom bar if the menu isn't already visible.

Comment (hockeychaos) [inline CoreScriptsRoot/Modules/Settings/SettingsHub.lua]: so how are we getting to this code then?

Comment (kyle-emmerich) [inline CoreScriptsRoot/Modules/Settings/SettingsHub.lua]: This callback always runs. :( I was going to submit another PR shortly after this that refactors it so that it runs only while the menu is visible and runs it once when the menu becomes visible.

Comment (kyle-emmerich) [inline CoreScriptsRoot/Modules/Settings/SettingsHub.lua]: But I could just do that now and put it all in one PR if you'd like.

Comment (hockeychaos) [inline CoreScriptsRoot/Modules/Settings/SettingsHub.lua]: yeah lets do the one
